src/database-layer/db.ts

```diff
@@ -166,6 +167,26 @@ export function registerTransactionLockNamespace(
 	transactionLockNamespaceMap[namespaceKey] = namespaceId;
 }
 
+const maybePrepareCache: {
+	[sqlHash: string]: number;
+} = {};
+const getPreparedName = (sql: Sql): string | undefined => {
+	if (env.db.prepareAfterN === false) {
+		return;
+	}
+	const sqlHash = stringHash(sql).toString(36);
+	if (env.db.prepareAfterN === true) {
+		return sqlHash;
+	}
+
+	maybePrepareCache[sqlHash] ??= 0;
+	if (maybePrepareCache[sqlHash] > env.db.prepareAfterN) {
+		return sqlHash;
+	}
+	maybePrepareCache[sqlHash]++;
+	return;
```

Comment:
Is the empty return necessary here when the function just ends?

Reply:
It's not necessary, it's just a remnant from a previous version - removed